server/api/contest_handler.py

```diff
@@ -0,0 +1,56 @@
+import json
+from flask import jsonify, Blueprint, request
+from api.models import contest_table, db
+from datetime import date
+contest_handler = Blueprint('contest_new_handler', __name__)
+
+
+@contest_handler.route('/create_new_contest', methods=['POST'])
```

Comment:
```suggestion
@contest_handler.route('/contest', methods=['POST'])
```

POST is dedicated to creating so naming it just `/contest/ would suffice